operators/advanced_network/advanced_network/manager.cpp

```diff
@@ -260,4 +260,67 @@ uint16_t Manager::get_num_rx_queues(int port_id) const {
   return cfg_.ifs_[port_id].rx_.queues_.size();
 }
 
+Status Manager::get_rx_burst(BurstParams** burst, int port_id) {
+  // Check if the port_id is valid
+  if (port_id < 0 || port_id >= static_cast<int>(cfg_.ifs_.size())) {
+    HOLOSCAN_LOG_ERROR("Invalid port_id {} provided to get_rx_burst", port_id);
+    return Status::INVALID_PARAMETER;
+  }
+
+  const auto& queues = cfg_.ifs_[port_id].rx_.queues_;
+  if (queues.empty()) {
+    // No queues configured for this port
```

Comment:
should I log error for requesting a port with no rx queue?

Reply:
I think that's a good idea